applications/matter_bridge/doc/matter_bridge_description.rst

```diff
@@ -103,6 +103,10 @@ For information about how to upgrade the device firmware using a PC or a smartph
 
 .. _matter_bridge_app_bridged_support:
 
+There is an additional Functionality which allow matter bridge device to work as smart plug device.
+Thanks to it, we can integrate bridge functionality in a real device which is smart plug, and do not have create a bridge device as standalone device.
+It is optional Feautre, and can be enabled by passing below building commands.
```

Comment:
Below is where? :) I would suggest putting here a ref link to the configuring section.
```suggestion
It is an optional feature, and can be enabled by passing below building commands.
```

Reply:
This comment should not be resolved as well. You write "below command", but below there is end of paragraph. I would suggest creating a ref build to the configuring section you have added and using it here as a link to direct user there.